devel/libjodycode/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        jbruchon libjodycode 3.1 v
+revision            0
+checksums           rmd160  521a8c7eada622e8569e5bd3d6add6cf1fb40a99 \
+                    sha256  7e07cad7931602b6de2130d0ad82cc964d2f5dae17853e7e54f50b91fc205960 \
+                    size    27359
+
+platforms           darwin
+categories          devel
+license             MIT
+maintainers         nomaintainer
+description         Library for imagepile, jdupes, winregfs, zeromerge.
+long_description    ${name} is a software code library containing code \
+                    shared among several of the programs written by \
+                    Jody Bruchon such as imagepile, jdupes, winregfs, \
+                    and zeromerge.
+
+# The following patch adds the absolute path to -install_name, along
+# with version numbers to -compatibility_version and -current version.
+# This patch also renames the compiled library and symlinks to .dylib
+# rather than .so.
+patchfiles          Makefile.patch
+
+use_configure       no
+
+build.args          PREFIX=${prefix} \
```

Comment:
It might make sense to use the `makefile` portgroup.

Reply:
Didn't know about the `makefile` portgroup—thank you!